wrappers/swift/DiagnosticDataViewer.swift

```diff
@@ -0,0 +1,85 @@
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/// Wrapper class over `ODWDiagnosticDataViewer` representing Diagnostic Data Viewer Hook.
+public class DiagnosticDataViewer {
```

Comment:
Do we expect subclasses of this class? If not, we should mark this as `final` to prevent unexpected behaviors if someone chooses to subclass.